driver-core/src/main/com/mongodb/connection/SimpleCommandMessage.java

```diff
@@ -31,9 +31,7 @@
 import static com.mongodb.ReadPreference.primary;
 
 /**
- * A command message that uses OP_QUERY to send the command.
- *
- * @mongodb.driver.manual ../meta-driver/latest/legacy/mongodb-wire-protocol/#op-query OP_QUERY
+ * A command message that uses OP_MSG or OP_QUERY to send the command.
  */
 class SimpleCommandMessage extends CommandMessage {
```

Comment:
I don't think this class is pulling enough weight anymore.  It's essentially the same as SplittablePayloadCommandProtocol but without the payload support.  I think the two classes should be merged together into CommandMessage, with an optional payload, and the two subclasses deleted.

Reply:
Done